$NetBSD: patch-util_grub.d_10__netbsd.in,v 1.1 2012/07/25 13:35:38 gregoire Exp $

Make sure that /netbsd is the main boot menu entry.

--- util/grub.d/10_netbsd.in.orig	2012-07-25 13:23:50.000000000 +0000
+++ util/grub.d/10_netbsd.in
@@ -147,7 +147,7 @@ pattern="^ELF[^,]*executable.*statically
 submenu_indentation=""
 
 is_first_entry=true
-for k in $(ls -t /netbsd*) ; do
+for k in /netbsd $(ls -t /netbsd?*) ; do
   if ! grub_file_is_not_garbage "$k" ; then
     continue
   fi
